planet/order_request.py

```diff
@@ -86,6 +87,9 @@ def build_request(name: str,
     if tools:
         details['tools'] = tools
 
+    if stac:
+        details['metadata'] = {'stac': {}}
```

Comment:
This is an aside, but I notice that the metadata object is not well documented here: https://developers.planet.com/docs/orders/reference/#tag/Orders/operation/createOrder.  I'll create an upstream ticket looking into why that part of the spec doesn't get documented.